meson.build

```diff
@@ -249,6 +249,7 @@ if build_daemon
   endif
 endif
 libm = cc.find_library('m', required: false)
+efivar = cc.find_library('efivar', required: false)
```

Comment:
host_machine.system() == 'freebsd' and then drop the `required: false` i think

Reply:
Done.